docs/src/content/docs/configuration.mdx

```diff
@@ -225,3 +225,15 @@ Exclude the `<details>` HTML element.
 
 Collapse whitespace.
 When `minify.whitespace` is set to `true`, all whitespace — including new lines — is collapsed to a single space character.
+
+### `pageSeparator`
+
+**Type:** `string`  
+**Default:** `\n\n`
+
+The separator used when concatenating page entries together.
+
+```ts
+pageSeparator: `\n----------\n`,
+```
```

Comment:
Tiny nit on the docs formatting here just to underline that the default is a string and use a regular string literal instead of a template literal in the example:

````suggestion
**Default:** `"\n\n"`

The separator used when concatenating page entries together.

```ts
pageSeparator: '\n----------\n',
```
````